[TF-37410] docs: differentiate Stacks and Workspaces in actions invocation docs (#2431)
hashicorp/web-unified-docs: content/terraform/v1.15.x/docs/language/invoke-actions.mdx

### Invoke an action from the CLI

> Direct action invocations using the `-invoke` flag are only supported for Workspaces. Stacks execute runs exclusively in HCP Terraform and do not support local CLI invocations. To invoke an action in a Stack, configure an [`action_trigger` block](#trigger-an-action-when-applying-a-configuration) in your resource lifecycle configuration instead.

Reviewer: ```suggestion
You can only target specific actions using the `-invoke` flag when your configuration is in a workspace. Stacks execute runs exclusively in HCP Terraform and do not support local CLI invocations. To invoke an action in a Stack, configure an [`action_trigger` block](#trigger-an-action-when-applying-a-configuration) in your resource lifecycle configuration instead.
```
I think we should move this paragraph to the [Requirements](https://developer.hashicorp.com/terraform/language/invoke-actions#requirements) section between the first paragraph and the HCP Terraform-specific requirements section. We can replace the paragraph here with something like "You can't directly invoke an action from a Stacks environment. Refer to Requirements for more information."

## View actions in HCP Terraform

> How you view action runs in HCP Terraform depends on whether you are using Workspaces or Stacks.

Reviewer: ```suggestion
You can view action runs in workspaces and Stacks.
```
